src/Iris/Cli/Interactive.hs

```diff
@@ -46,3 +49,8 @@ interactiveModeP = nonInteractiveP <|> pure Interactive
         [ Opt.long "no-input"
         , Opt.help "Enter the terminal in non-interactive mode"
         ]
+
```

Comment:
Since it's a part of exported API, let's add documentation to this function with the `@since x.x.x.x` annotation

Reply:
what version should it be? 0.0.0.1?

Reply:
It should be `x.x.x.x`. I run a CLI command before the release to replace `x.x.x.x` with the actual version when the resulting version is known.